### PR TITLE
Test for detection status file change

### PR DESCRIPTION
**Pull request by bruntib**

### CODE REVIEW

File: tests/functional/detection_status/test_detection_status.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 import shared
 
+from codeCheckerDBAccess.ttypes import Encoding
```
Comment (whisperity) [inline tests/functional/detection_status/test_detection_status.py]: Please keep import order according to [CONTRIBUTING.md](https://github.com/Ericsson/codechecker/blob/master/CONTRIBUTING.md).

```py
import json
import os
import unittest

import shared
from codeCheckerDBAccess.ttypes import Encoding

from libtest import codechecer
from libtest import env
```

File: tests/functional/detection_status/test_detection_status.py [hunk 2]
```diff
@@ -131,14 +125,25 @@ def test_same_file_change(self):
...
+
+                file_content = self._cc_client.getSourceFileData(
+                    report.fileId,
+                    True,
+                    Encoding.DEFAULT).fileContent
+
+                self.assertEqual(
+                    file_content,
+                    self.sources[1],
+                    'Resolved bugs should be shown with the new file content.')
```
Comment (whisperity) [inline tests/functional/detection_status/test_detection_status.py]: This is a human string, it should go inbetween `"`s and not `'`s.

Comment (dkrupp) [inline tests/functional/detection_status/test_detection_status.py]: unresolved bugs should be shown with the new file content.

File: tests/functional/detection_status/test_detection_status.py [hunk 3]
```diff
@@ -163,11 +168,23 @@ def test_same_file_change(self):
                     shared.ttypes.DetectionStatus.UNRESOLVED:
                 self.assertIn(report.bugHash,
                               ['cbd629ba2ee25c41cdbf5e2e336b1b1c'])
+
+                file_content = self._cc_client.getSourceFileData(
```
Comment (dkrupp) [inline tests/functional/detection_status/test_detection_status.py]: should we also test for resolved bugs too?

**Comment by gyorb** [comment]
Please resolve the merge conflicts.

File: tests/functional/detection_status/test_detection_status.py (hunk 3 above)
Comment (bruntib) [reply tests/functional/detection_status/test_detection_status.py]: Done

**Pull request merged**